apps/condo/domains/billing/components/BillingPageContent/ContextProvider.ts

```diff
@@ -0,0 +1,18 @@
+import { createContext, useContext } from 'react'
+
+import type { BillingIntegrationOrganizationContext, AcquiringIntegrationContext } from '@app/condo/schema'
+
+type IBillingAndAcquiringContexts = {
+    billingContext: BillingIntegrationOrganizationContext
+    acquiringContext: AcquiringIntegrationContext
```

Comment:
Didn't find usages of acquiring context. Are both contexts really needed to be placed in the same react context?
<img width="753" alt="image" src="https://user-images.githubusercontent.com/25384290/234467989-572797cc-f235-4ede-bb90-b9c3232afaec.png">

Reply:
Yep, added them in case of need in payments / accruals tab. However, payments somehow does not use that. But left them both in case they'll be needed